packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/bedrock/invoke-model.ts

```diff
@@ -207,11 +237,13 @@ export class BedrockInvokeModel extends sfn.TaskStateBase {
         Body: this.props.body?.value,
         Input: this.props.input?.s3Location ? {
           S3Uri: `s3://${this.props.input.s3Location.bucketName}/${this.props.input.s3Location.objectKey}`,
-        } : undefined,
+        } : this.props.inputPath ? { S3Uri: this.props.inputPath } : undefined,
```

Comment:
i see the `S3Uri` has a specific format, should we check if `inputPath` and `outputPath` matches that format?

Reply:
current s3 uri is limiting and enforcing that format., this props is to allow user to break out of that specific format and allow value usage from prompt which would be at step-function runtime i suppose which might be difficult to check here same way we can't limit our policy to that specific input hence used `*` as above.